asdf/tests/test_api.py

```diff
@@ -401,22 +401,16 @@ def test_auto_inline_masked_array(tmpdir):
         assert len(list(af.blocks.internal_blocks)) == 2
 
 
-def test_auto_inline_large_value(tmpdir):
-    outfile = str(tmpdir.join('test.asdf'))
-    arr = np.array([2**52 + 1] + [k for k in range(constants.DEFAULT_AUTO_INLINE)])
-    tree = {"array": arr}
-
-    with asdf.AsdfFile(tree) as af:
-        af.write_to(outfile)
-        assert len(list(af.blocks.inline_blocks)) == 0
-        assert len(list(af.blocks.internal_blocks)) == 1
+@pytest.mark.parametrize("val", [constants.MAX_NUMBER+1, constants.MIN_NUMBER-1])
+def test_auto_inline_large_value(val):
```

Comment:
I removed this test.  The original test made sure that for short arrays, but large numbers the 52 limit was enforced.  Since that limit is removed, this test no longer applies.